src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from '../types/Person';
+import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [activePerson, setActivePerson] = useState<Person | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    setError(null);
+
+    getPeople()
```

Comment:
Ensure that the `getPeople` function is correctly implemented and returns the expected data format. Verify that it handles network errors and edge cases properly.